auth-server/src/services/auth.ts

```diff
@@ -107,14 +108,14 @@ export default class AuthService {
 
 	public async ChangePassword(token: string, oldPassword: string, newPassword: string) {
 		if (oldPassword === newPassword) {
-			throw new Error('Old password cannot be same as new password')
+			throw new UserInputError('Old password cannot be same as new password')
 		}
 
 		// verify a token symmetric - synchronous
 		const decoded = jwt.verify(token, publicKey)
 
 		if (isNaN(decoded.sub)) {
-			throw new Error('Invalid user id')
+			throw new ForbiddenError('Invalid user id in token')
```

Comment:
Shouldn't this rather be a `AuthenticationError` because it comes from the token?